src/Definitions/OrderType.php

```diff
@@ -9,6 +9,9 @@ class OrderType extends Definition
         PRICE_ASCENDING = 'price ASC',
         PRICE_DESCENDING = 'price DESC',
         ALPHABETICAL = 'label ASC',
-        TOP_SELLERS_FIRST = 'salesfrequency DESC',
-        NEWEST_FIRST = 'dateadded DESC';
+        REVERSE_ALPHABETICAL = 'label DESC',
+        TOP_SELLERS_FIRST = 'salesfrequency dynamic DESC',
```

Comment:
Rename this to `TOP_SELLERS_DYNAMIC_FIRST`/`LAST`, and re-add `TOP_SELLERS_FIRST`/`LAST`.